config/management/operational/src/command.rs

```diff
@@ -26,10 +28,18 @@ pub enum Command {
     CreateValidator(crate::governance::CreateValidator),
     #[structopt(about = "Create a new validator operator account")]
     CreateValidatorOperator(crate::governance::CreateValidatorOperator),
+    #[structopt(about = "Extract peer information from an x25519 PrivateKey file")]
+    ExtractPeerFromFile(crate::keys::ExtractPeerFromFile),
```

Comment:
might be useful to better define what peer information is -- namely the information required for configuring a trusted network identity.